de.metas.contracts/src/main/java/de/metas/flatrate/process/C_Flatrate_Term_Change.java

```diff
@@ -66,7 +66,7 @@ protected String doIt()
 		
 		final I_C_Flatrate_Term currentTerm = InterfaceWrapperHelper.create(getCtx(), getRecord_ID(), I_C_Flatrate_Term.class, get_TrxName());
 
-		Services.get(IContractChangeBL.class).cancelContract(currentTerm, changeDate);
+		Services.get(IContractChangeBL.class).cancelContract(currentTerm, changeDate, true);
```

Comment:
I just wanted to preserve the old behaviour. By default is to close invoice candidates.